catalog-scripts/idp-catalog-wizard-bitbucket.py

```diff
@@ -0,0 +1,175 @@
+import argparse
```

Comment:
Let's add/update readme for this script
And also add a curl command to download this file